apps/portal-api/src/modules/users/users.profile.app.ts

```diff
@@ -39,4 +48,55 @@ export const usersProfileApp = {
 
     return mappedUser;
   },
+  requestTransferPersonalSpace: async (context, newEmail: string) => {
+    if (!isValidEmail(newEmail)) {
+      throw new Error(ErrorCode.InvalidEmail);
+    }
+    const existingPersonalSpace = await loadOrganizationBy({
+      name: newEmail,
+      personal_space: true,
+    });
+    if (!existingPersonalSpace) {
+      throw new Error(ErrorCode.PersonalSpaceMustAlreadyExist);
+    }
+    const newUser = await loadUserByOrganization(existingPersonalSpace.id);
+    await sendMail({
+      to: newEmail,
+      template: 'request_transfer_personal_space',
+      params: {
+        recipientName: `${newUser[0].first_name} ${newUser[0].last_name}`,
+        recipientId: newUser[0].id,
+        previousUserId: context.user.id,
+        previousUserEmail: context.user.email,
+        previousUserName: `${context.user.firstname} ${context.user.lastname}`,
+      },
+    });
```

Comment:
If you remove the `select('*')` from the loadUserByOrganization you can do that :

```suggestion
const newUser = await loadUserByOrganization(existingPersonalSpace.id)
await loadUserByOrganization(existingPersonalSpace.id)
  .select('id', 'first_name', 'last_name', 'email')
  .first();
.first();

await sendMail({
  to: newEmail,
  template: 'request_transfer_personal_space',
  params: {
    recipientName: `${newUser.first_name} ${newUser.last_name}`,
    recipientId: newUser.id,
    previousUserId: context.user.id,
    previousUserEmail: context.user.email,
    previousUserName: `${context.user.firstname} ${context.user.lastname}`,
  },
});
```